config/default.yml

```diff
@@ -114,7 +114,7 @@ Metrics/BlockLength:
   Enabled: true
   VersionAdded: 'TBD'
```

Comment:
We can use `'<<next>>'` like [`rubocop-rails` does](https://github.com/rubocop/rubocop-rails/commit/0e06a0ee424b3ca7f7f83b25d9335116e6d376ec#diff-3653213e3a153018a75cc7b2536b1d0217a6d8d75188d71cf45441812f789698R881)?

Reply:
Do we even need this? 🤔